code/modules/projectiles/ammo_datums.dm

```diff
@@ -156,6 +156,8 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 		var/obj/projectile/new_proj = new /obj/projectile(main_proj.loc)
 		if(bonus_projectiles_type)
 			new_proj.generate_bullet(GLOB.ammo_list[bonus_projectiles_type]) //No bonus damage or anything.
```

Comment:
Time to remove this comment since you ARE now applying bonus damage.

Reply:
Hugh I think you're mistaken, I tested the combat shotgun with buckshot and the additional projectiles gained 8 damage from a base of 40 from a Barrel Charger